lib/Document.php

```diff
@@ -125,13 +125,13 @@ public function download(?bool $postProcess = true, ?string $encoding = null): s
         if (!($this->contentType === ContentType::XLSX || $this->contentType === ContentType::PDF)) {
             if (!is_null($encoding) && !in_array(strtoupper($encoding), mb_list_encodings())) {
                 $encoding = null;
-            } else {
+            } else if (is_null($encoding)) {
                 $encodings = ['UTF-8'];
                 if ($response->hasHeader('content-type')) {
                     $httpContentType = $response->getHeader('content-type');
                     $parsedHeader = \GuzzleHttp\Psr7\Header::parse($httpContentType);
                     if (isset($parsedHeader[0]['charset'])) {
-                        array_unshift($encodings, $parsedHeader[0]['charset']);
+                        array_unshift($encodings, str_replace("Cp1252", "ISO-8859-1", $parsedHeader[0]['charset']));
```

Comment:
could you please add a comment here explaining why you're doing the `str_replace`?

Reply:
done at https://github.com/jlevers/selling-partner-api/pull/341/commits/a2fb87dcc387232c91cc3e711ac530c72b768cdf